configs/rollup.config.js

```diff
@@ -126,7 +126,10 @@ const webConfigs = [
     external: ['react', 'react-dom', '@stylexjs/stylex'],
     input: require.resolve('../packages/react-strict-dom/src/dom/index.js'),
     output: {
-      file: path.join(__dirname, '../packages/react-strict-dom/dist/dom.js'),
+      file: path.join(
+        __dirname,
+        '../packages/react-strict-dom/dist/dom/index.js'
+      ),
```

Comment:
What about the www build? That is still being written to `dist/dom.www.js`; how is internal use going to work if the types are being picked up from the new flow files but they have references to the `@stylexjs/stylex` package instead of `stylex`?

Reply:
I will be transforming the imports while copying the files over during sync.